DataFormats/Detectors/FIT/FT0/include/DataFormatsFT0/Digit.h

```diff
@@ -86,22 +86,25 @@ struct Triggers {
 };
 
 struct Digit {
-  o2::dataformats::RangeRefComp<5> ref;
+  o2::dataformats::RangeRefComp<8> ref;
 
   Triggers mTriggers;               // pattern of triggers  in this BC
   o2::InteractionRecord mIntRecord; // Interaction record (orbit, bc)
-
+  int mEventID;
```

Comment:
The digit may be contributed by multiple events, right? What is the meaning of this mEventID? As far as I see, it will get the ID of the last event contributing to this this digit. Is this what you need?

Reply:
@AllaMaevskaya And what about this?

Reply:
I need this to how many BCs after "interaction" one were recorded by "slow" tail. This is very convenient to analyse  pile-up, I see number of BCs per event.

Reply:
You just register the ID of the event which triggered the creation of given digit. Is this what you want?

Reply:
As I see in digitzer printout after "main" interaction  FT0 registered data in few next bunches.
![image](https://user-images.githubusercontent.com/20040756/78228882-882e2800-74cf-11ea-908d-f871872b2fcf.png). This tail can generate "false" triggers

Reply:
did not fully understand but ok.